hail/python/hail/backend/service_backend.py

```diff
@@ -382,7 +383,7 @@ async def _rpc(self,
                     raise
 
             with timings.step("parse status"):
-                if status['n_succeeded'] != 1:
+                if status['n_succeeded'] == 0:
```

Comment:
Can you explain this change to me? I would have thought it should have been `if status['n_failed'] > 0`

Reply:
you're right, will change this